UnityProject/Assets/Scripts/Items/Science/Implants/ReviverImplant.cs

```diff
@@ -15,6 +16,11 @@ public class ReviverImplant : BodyPartFunctionality
 
 		private float lastTrigger = 0;
 
+		public bool isEMPVunerable = false;
```

Comment:
don't tell me you just copy pasted these everywhere

Reply:
only the places that Used EMP stuff